src/main/java/org/jooq/lambda/Seq.java

```diff
@@ -3397,6 +3397,30 @@ public R next() {
     }
 
     /**
+     * Chunk a Stream based on a predicate
+     * <p>
+     * <code><pre>
+     * // ((1,1,2), (2), (3,4))
+     * Seq.of(1, 1, 2, 2, 3, 4).chunked(n -> n %2 == 0)
```

Comment:
Hmm, I think we should exclude the chunk boundaries by default if using a predicate, and include them only explicitly. When including, the question is whether the boundary should belong to:

1. The ending chunk `((1, 1, 2), (2), (3, 4))`
2. The beginning chunk `((1, 1), (2), (2, 3), (4))`
3. Both chunks (?)
4. No chunk

Given that there is no clear preference between 1/2 and maybe even 3, the default must be 4. So, an enum might be a reasonable additional argument for this, where "no chunk" would be the default.